test/src/test/java/com/distkv/dst/test/supplier/BaseTestSupplier.java

```diff
@@ -26,7 +26,7 @@ public void setupBase(Method method) throws InterruptedException {
         method.getDeclaringClass(), method.getName()));
     rpcServerPort = (Math.abs(new Random().nextInt() % 10000)) + 10000;
     TestUtil.startRpcServer(rpcServerPort);
-    TimeUnit.SECONDS.sleep(1);
+    TimeUnit.SECONDS.sleep(2);
```

Comment:
What's the meaning of this change? @kairbon 
After the master-slave synchronization added, the server starts slower?

Reply:
Yes, I mean that.